portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,36 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
Don't use wildcard imports. They come with all kinds of drawbacks around potential naming conflicts.

See e.g. https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad